cookbooks/muparser_temperature_example/muparser-temperature-example.prm

```diff
@@ -64,6 +64,11 @@ end
 # Function expression asks if the depth is less than the lithosphere depth and
 # then refinement level = 5, else  refinement level = 0
 
+# Note that if the refinemet level in the else statement (e.g., 0 here) in the 
+# function expression is lower than initial global refinement level, the lower 
+# 850 km of the mesh is 1 level down coarsening from initial global refinement level. 
```

Comment:
You could, but I think it would still want to explain how this value is specifically chosen to match the initial global refinement.  The way in which different parameter choices combine is actually quite confusing.